dagql.ebnf

```diff
@@ -0,0 +1,26 @@
+(* we possibly do not want to include sub-queries in FROM *)
+query = "SELECT" (attr ("," attr)* | "*")
+        "FROM" ("EDGES" | "NODES" | query)
```

Comment:
Should we make it explicit that whitespace is expected between these keywords, and that queries might be parenthesized?

Reply:
I don't think we need to make the first part explicit, but I can make it so sub-queries can be parenthesized.

Reply:
One thing from me is make sure we know how handle the query for NODES or EDGES Since they have different props.

Also, can DAQ be weighted?

Reply:
michaelangelo, that's not really related to the EBNF? also, what do you mean by DAQ?

Reply:
Typo DAG not DAQ lol

Reply:
Typo DAG not DAQ lol